src/components/Canvas/Editor.tsx

```diff
@@ -1484,6 +1534,10 @@ export default class Editor extends EventDispatcher {
             pixelIndex.rowIndex,
             pixelIndex.columnIndex,
           );
+          this.fillMissingIndices(
+            this.mouseMoveWorldPos,
+            this.previousMouseMoveWorldPos,
+          );
```

Comment:
Seems like you mispositioned the parameters
```suggestion
          this.fillMissingIndices(
            this.previousMouseMoveWorldPos,
            this.mouseMoveWorldPos,
          );
```

Reply:
fixed